scipy/special/cephes/hyp2f1.c

```diff
@@ -83,6 +83,43 @@ static double hyt2f1(double a, double b, double c, double x, double *loss);
 static double hys2f1(double a, double b, double c, double x, double *loss);
 static double hyp2f1ra(double a, double b, double c, double x,
 		       double *loss);
+static double pochhammer(double x, double n);
+static double factorial(double k);
+static double hyp2f1_indeterminate(double a, double b, double x);
+
+/*
+    See http://mathworld.wolfram.com/PochhammerSymbol.html
+*/
+static double pochhammer(double x, double n)
+{
+    return gamma(x + n) / gamma(x);	
+}
+
+/*
+    See http://mathworld.wolfram.com/GammaFunction.html
```

Comment:
No need for separate function, please write in terms of gamma.

Reply:
Will do.